src/components/Header/Logo.test.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import Logo from './Logo';
+
+describe('Logo component', () => {
+  it('should include alt property', () => {
+    const { getByAltText } = render(<Logo src="src" />);
```

Comment:
Is there a reason we wouldn't access this off of `screen`?

Reply:
no reason, i overlooked it, updating it.